src/bin/uhyve.rs

```diff
@@ -126,7 +126,8 @@ struct UhyveArgs {
 	/// GDB server port
 	///
 	/// Starts a GDB server on the provided port and waits for a connection.
-	#[clap(short = 's', long, env = "HERMIT_GDB_PORT")]
+	/// If no port is entered, a random port will be chosen instead.
+	#[clap(short = 's', long, env = "HERMIT_GDB_PORT", num_args(0..=1), default_missing_value("0"))]
```

Comment:
I can't imagine a scenario where a random port is what we want. We could instead define a default port. The Wiki currently suggests 6677 ([which isn't even conflicting with major known software](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers)).

Reply:
If somebody ever has the situation to debug two Unikernels, manually assigned ports for each instance is probably what they want. And if somebody really wants a random port, first of all, please notify me because I'm curious on the use case, and second, just use `-s 0` as parameter.

So yes. Please just default to 6677 if no parameter was given.